examples/node_disk_failure.yaml

```diff
@@ -0,0 +1,17 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Why do we need this example? It seems to be the same one than with the pod except that it's targeting a node but this example is already covered in the node level example.

Reply:
I have created this example to show how to define a disk failure targeting a node, but I agree with you, it is useless 👍